cluster/src/main/java/com/hartwig/pipeline/calling/structural/gripss/GripssGermline.java

```diff
@@ -11,13 +17,57 @@
 
 public class GripssGermline extends Gripss {
 
+    public static final String GERMLINE_NAMESPACE = "gripss_germline";
+
+    private static final String GRIPSS_GERMLINE_FILTERED = ".gripss.filtered.germline.";
+    private static final String GRIPSS_GERMLINE_UNFILTERED = ".gripss.germline.";
+
     public GripssGermline(final GridssOutput gridssOutput, final PersistedDataset persistedDataset,
             final ResourceFiles resourceFiles) {
-        super(gridssOutput, persistedDataset, GripssConfiguration.germline(resourceFiles));
```

Comment:
Not sure why you re-refactored this to use abstract template methods again. I thought we wanted to reduce the code in the subclasses? Which is why I created the GripssConfiguration class to go for a more compositional approach. 

I don't really mind that much one way or the other, but would be good to decide what is the preferred approach and keep it consistent (for instance now Sage uses SageConfiguration over a bunch of template methods).